website/content/docs/concepts/workers.mdx

```diff
@@ -0,0 +1,94 @@
+---
+layout: docs
+page_title: Workers
+description: |-
+  Introduction to Boundary workers
+---
+
+# Workers
+
+## Overview
+Boundary's architecture consists of 2 main components:
+1. **Control Plane** - made up of controllers
+2. **Data Plane** - made up of workers
+
+**Controllers** are where users authenticate to, and contain Boundary's resources and permissions. In addition, controllers also communicate with external components 
```

Comment:
"Controllers are what users authenticate to through a client..."

Reply:
Will change